src/Acuminator/Acuminator.Analyzers/Refactorings/EventHandlerSignature/EventHandlerSignatureRefactoring.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Acuminator.Utilities.Common;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.Roslyn.Semantic;
+using Acuminator.Utilities.Roslyn.Syntax;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Acuminator.Analyzers.Refactorings.EventHandlerSignature
+{
+	[ExportCodeRefactoringProvider(LanguageNames.CSharp), Shared]
```

Comment:
@dropsonic Should the Name property of the attribute be set?

Reply:
@SENya1990 nope, it is not required